browser/src/Services/Completion/CompletionSelectors.ts

```diff
@@ -3,6 +3,7 @@
  *
  * Selectors are functions that take a state and derive a value from it.
  */
+import * as _ from "lodash"
```

Comment:
@feltech if you import just the specific functions you need rather than `_` from lodash this will help reduce our bundle size which is something we should aim for but probably don't get right in many situations

e.g.
```ts
import uniq from "lodash/uniq"
// or if type issues
import * as omit from "lodash/omit"
```